### ISSUE TITLE
Chromium not supported?

### ISSUE THREAD

**Issue by weyert** [flotwig/cypress-log-to-output#15]
I tried this plugin out for Cypress but it appears it's not supporting Chromium as I am getting the error when trying to use it:

```
Warning: An unsupported browser family was used, output will not be logged to console: chromium
```

Is this expected? I am using the Cypress docker image

**Comment by flotwig** [comment]
@weyert could you try upgrading to v1.1.2? just published a fix which should help your situation

**Comment by nmkettler** [comment]
@flotwig I'm still receiving that error as well for electron, even after upgrading to v1.1.2. I am setting the browser via the module api:

```
cypress
  .run({
    reporter: 'junit',
    browser: 'electron',
    headless: true,
    project: './',
    detached: false,
    config: { baseUrl, video: false, record: false },
    env: {
      ...env,
      HOST_URL: baseUrl,
    },
  })
  .then(res => {
    console.log('cypress results');
    if (res.failures !== 0) process.exit(1);
  })
  .catch(err => {
    console.log(err);
    process.exit(1);
  });
```

**Comment by flotwig** [comment]
@nmkettler this module is not supported for Electron, see #2

**Comment by flotwig** [comment]
@weyert going to close this since it should have been fixed by v1.1.2, please open a new issue if it crops up again
